.github/workflows/release-charts.yml

```diff
@@ -0,0 +1,35 @@
+name: Release Charts
+
+on:
+  push:
+    paths:
+      - 'charts/**'
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Fetch history
+        run: git fetch --prune --unshallow
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      # See https://github.com/helm/chart-releaser-action/issues/6
+      - name: Install Helm
+        run: |
+          curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+          chmod 700 get_helm.sh
+          ./get_helm.sh
```

Comment:
@sonasingh46 . There are some [changes](https://github.com/openebs/charts/blob/bb67af5d9013e8d7c60630709915926143285177/.github/workflows/release.yaml#L29) with the release file given in openebs/charts. Is it intentional?

Reply:
Yes. This one is removed from helm3 as in helm3 there is no tiller. 
Further reading : 
https://helm.sh/docs/helm/helm_init/#helm-init